specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/examples/LocationExtensionTypes_List.json

```diff
@@ -0,0 +1,44 @@
+{
+  "parameters": {
+    "subscriptionId": "aaaaaaaaaa",
+    "location": "aaaaaa",
+    "api-version": "2021-05-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
```

Comment:
I assume extensions available in a given location is a superset of extensions across all the clusters in that location. And users will be able to select one of the extension on their cluster.

Reply:
Correct